psi4/share/psi4/databases/O24by5mb.py

```diff
@@ -0,0 +1,2858 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2021 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""
+| Database (O24) of interaction energies for small open-shell high-spin bimolecular complexes.
+| Geometries taken from https://gitlab.com/piotr.zuchowski/o24x5/-/tree/4ec3cae0546b6ae4f4f2cf28482cd164c8323cc6.
+| Reference interaction energies taken from J. Chem. Phys. 154, 134106 (2021) https://doi.org/10.1063/5.0043793.
+| Variant with ghosted hydrogen as midbonds in the COM.
+| 
+| no | name           | subset
+|  1 | CN   - He      | DD
+|  2 | NH   - He      | DD
+|  3 | C2H3 - C2H4    | DD
+|  4 | O2   - H2      | DD
+|  5 | NH   - Ar      | DD
+|  6 | CN   - Ar      | DD
+|  7 | O2   - N2      | DD
+|  8 | H2O  - O2(sp)  | DD
+|  9 | O2   - O2      | DD
+| 10 | NH   - NH      | ED
+| 11 | CH2O - NH2     | ED
+| 12 | H2O  - Na      | ED
+| 13 | H2O  - OH      | ED
+| 14 | H2O  - O2H     | ED
+| 15 | Li   - NH3(gm) | ED
+| 16 | Li   - O2      | MX
+| 17 | CN   - H2      | MX
+| 18 | Li   - NH3(lm) | MX
+| 19 | H2O  - O2(gm)  | MX
+| 20 | Na   - Li      | MX
+| 21 | CO2  - O2      | MX
+| 22 | C2H3 - CO2     | MX
+| 23 | He*  - He*     | MX
+| 24 | HF   - CO+     | MX
+
+- **cp**  ``'on'``
+
+- **subset**
+  - ``'DD'`` dispersion-dominated systems
+  - ``'ED'`` electrostatically-dominated systems
+  - ``'MX'`` mixed-interaction systems
+
+"""
+
+import qcdb
+
+from psi4.driver import constants
+
+# <<< O24by5 Database Module >>>
+dbse = "O24by5mb"
```

Comment:
Is there no way to combine the two variants into a single file? The midbond positions could easily be added into the geometry database automatically (as we always seem to add `Gh(H) @ 0.0 0.0 0.0`), the only thing I'm not sure the runner knows how to do is to select a `GEOM` based on the `dbse` that's used to call this...

Reply:
maybe have the non-mb database file define the geometry strings in a dict, then strip the last two lines before feeding into Molecule constructor in GEOS (units, etc. can be first lines in the string), then the mb database imports the plain one, deep copies all the data structures and uses the full strings in GEOS.

Or, database files are pretty static, and these two are easy to diff to check sync. I can live with the duplication

Reply:
If it's fine by you, it's fine by me. I was just wondering if we were missing a trick there, but if it'd have to be hacked to avoid duplication, those few LoC are worth a more obvious solution.